orcid_service/models.py

```diff
@@ -192,10 +192,13 @@ def update_status(self, keys, status):
         """
         if type(keys) is not list:
             keys = [keys]
+        if type(status) is list:
```

Comment:
If you have fixed the pipeline, I would suggest to remove this from here given that we are not verifying every single data type that it cannot be (e.g., what if it is a dictionary?). If we really need to check this status, I would check if status is a string and if not fail with an error. What do you think?